tests/test_demographics.py

```diff
@@ -428,3 +428,24 @@ def test_alt_phone_removal(self):
                 self.fail(
                     'unexpected telecom system: {}'.format(item['system']))
 
+    def test_deceased_undead(self):
+        "Confirm we can remove time of death via deceasedBoolean: False"
+        self.promote_user(role_name=ROLE.PATIENT)
+        d_audit = Audit(
+            user_id=TEST_USER_ID, subject_id=TEST_USER_ID, context='user',
+            comment="time of death for user {}".format(TEST_USER_ID))
+        with SessionScope(db):
+            db.session.add(d_audit)
+            db.session.commit()
+        self.test_user, d_audit = map(db.session.merge, (self.test_user, d_audit))
+        self.test_user.deceased = d_audit
+        self.login()
+        data = {"resourceType": "Patient",
+                'deceasedBoolean': False}
+        rv = self.client.put(
+            '/api/demographics/%s' % TEST_USER_ID,
```

Comment:
just curious, would it make sense to use `url_for()` in tests...?

Reply:
alters the setup - one could certainly make the case for it, but we're currently doing it this way.  /shrug